### PR TITLE
Feature/153 add support for hapic file for all framework

### PR DESCRIPTION
**Pull request by inkhey**
related to #153

### CODE REVIEW

File: hapic/ext/aiohttp/context.py [hunk 1]
```diff
@@ -133,6 +134,17 @@ def get_request_parameters(self, *args, **kwargs) -> RequestParameters:
...
+    def get_file_response(
+        self,
+        file_response: HapicFile,
+        http_code: int
+    ) -> "Response":
+        if file_response.file_path:
+            # FIXME - G.M - 2019-03-26 - handle others headers in HapicFile
+            return web.FileResponse(path=file_response.file_path)
+        else:
+            raise NotImplementedError()
```
Comment (buxx) [inline hapic/ext/aiohttp/context.py]: Don't forgot to test all case of  ̀HapicFile` before MR opening ?

File: hapic/ext/aiohttp/context.py [hunk 2]
```diff
@@ -133,6 +134,17 @@ def get_request_parameters(self, *args, **kwargs) -> RequestParameters:
 
         raise WorkflowException("Unable to get aiohttp request object")
 
+    def get_file_response(
+        self,
+        file_response: HapicFile,
+        http_code: int
+    ) -> "Response":
+        if file_response.file_path:
```
Comment (buxx) [inline hapic/ext/aiohttp/context.py]: Don't forgot to test all case of ̀HapicFile` before MR opening.

File: hapic/ext/bottle/context.py [hunk 3]
```diff
@@ -64,6 +65,15 @@ def get_request_parameters(self, *args, **kwargs) -> RequestParameters:
             files_parameters=files_parameters,
         )
 
+    def get_file_response(
+        self, file_response: HapicFile, http_code: int
+    ) -> bottle.HTTPResponse:
+        if file_response.file_path:
```
Comment (buxx) [inline hapic/ext/bottle/context.py]: Don't forgot to test all case of ̀HapicFile` before MR opening.

Comment (inkhey) [reply hapic/ext/bottle/context.py]: Done.

File: hapic/ext/aiohttp/context.py (hunk 2 above)
Comment (inkhey) [reply hapic/ext/aiohttp/context.py]: Done.

**Comment by inkhey** [comment]
@buxx support and tests was added for simple filepath hapicFile. Full support of all hapic file case will be handled in #171.

**Pull request merged**